core/types/calls.cc

```diff
@@ -4107,6 +4141,7 @@ class T_Enum_tripleEq : public IntrinsicMethod {
 const vector<Intrinsic> intrinsics{
     {Symbols::T(), Intrinsic::Kind::Singleton, Names::untyped(), &T_untyped},
     {Symbols::T(), Intrinsic::Kind::Singleton, Names::must(), &T_must},
+    {Symbols::T(), Intrinsic::Kind::Singleton, Names::mustBecause(), &T_must_because},
```

Comment:
You can likely get away with modifying `T_must` to be aware of whether it's handling `T.must` or `T.must_because`, and then use 
```suggestion
    {Symbols::T(), Intrinsic::Kind::Singleton, Names::mustBecause(), &T_must},
```

which cuts down on some duplication if you're interested.